docs/rst/tutorials/aml_pipeline.rst

```diff
@@ -1,24 +1,452 @@
+    print(batchLearner.unionModel[0].ucs)
+
+
+This displays the following information::
+
+    Union model size: 10409
+    Size spectrum:
+    L spectrum:
+      L 1 atoms 1496
+      L 2 atoms 27
+      L 3 atoms 193
+      L 4 atoms 286
+      L 5 atoms 509
+      L 6 atoms 637
+      L 7 atoms 755
+      L 8 atoms 959
+      L 9 atoms 992
+      L 10 atoms 938
+      L 11 atoms 852
+      L 12 atoms 713
+      L 13 atoms 572
+      L 14 atoms 362
+      L 15 atoms 254
+      L 16 atoms 194
+      L 17 atoms 167
+      L 18 atoms 110
+      L 19 atoms 75
+      L 20 atoms 50
+      L 21 atoms 54
+      L 22 atoms 29
+      L 23 atoms 33
+      L 24 atoms 24
+      L 25 atoms 18
+      L 26 atoms 23
+      L 27 atoms 13
+      L 28 atoms 13
+      L 29 atoms 7
+      L 30 atoms 7
+      L 31 atoms 6
+      L 32 atoms 6
+      L 33 atoms 8
+      L 34 atoms 6
+      L 35 atoms 1
+      L 36 atoms 4
+      L 37 atoms 5
+      L 38 atoms 1
+      L 39 atoms 2
+      L 40 atoms 3
+      L 41 atoms 2
+      L 42 atoms 1
+      L 44 atoms 2
+    Some random atoms:
+    bitarray({264})
+    bitarray({1184, 1568, 1351, 1512, 329, 303, 1136, 595, 596, 215, 125, 574})
+    bitarray({222})
+    bitarray({320, 1217, 1568, 357, 1134, 496, 1425, 1330, 244, 1268, 1052})
+    bitarray({1568, 129, 1220, 1535, 1165, 1293, 271, 592, 597, 1304, 217, 1437, 351})
+    bitarray({992, 513, 1568, 1029, 1194, 491, 939, 397, 653, 655, 1100, 690, 243, 1235, 1364, 1078, 539, 157})
+    bitarray({582})
+    bitarray({164})
+    bitarray({1568, 1134, 1044, 398})
+    bitarray({746})
+    Largest atom
+    bitarray({1280, 1408, 1160, 1162, 651, 1168, 657, 659, 1050, 1568, 289, 162, 425, 554, 1193, 1323, 302, 1198, 321, 970, 1357, 206, 1102, 208, 593, 1107, 1108, 215, 1113, 218, 348, 483, 996, 1251, 359, 1386, 620, 1132, 1133, 1136, 1137, 1267, 244, 509})
+
+
+Plotting results
+^^^^^^^^^^^^^^^^
 
-    Not available yet.
+For classification problems with square images the viewer module provides some functions to plot atoms.
+It simply maps back constant indices to their position and color on an image.
+It then uses ``matplotlib`` to output the results.
 
 
-Background
-**********
+.. code-block:: python
 
-.. note::
+    import viewer as av
 
-    TODO
+    theViewer = av.atomViewer_bw_sq_image(
+        cmanager=model.cmanager,
+        min_size=2,
+        scale=10,
+        atoms_per_board=[10, 10],
+    )
+    theViewer.save(f"RESULTS/MNIST_IMG_{targetDigit}_{i}", batchLearner.unionModel)
+    theViewer.save(f"RESULTS/MNIST_IMG_model_{targetDigit}_{i}", model.atomization)
 
-Prerequisites
-*************
+As an example, below we show 100 atoms in ``model.atomization``.
+Black and white pixels indicate that the constant is present in the atom; red, that both constants are present; and gray, that none is present.
 
-.. note::
+.. image:: /rst/figures/tutorials/example_atoms_mnist.png
+   :width: 400
+   :alt: A hundred atoms from the atomization of MNIST.
+   :align: center
```

Comment:
I'm missing another section for inference, I mean, given an image from MNIST dataset not used for training, infer if it belongs to that class or not.